src/consts.h

```diff
@@ -14,7 +14,10 @@
 // General Constants
 //------------------
 
-#define   VERSION            51011                                             //(5.1.011)
+// Update VERSION and SEMVERSION Simultaneously 
+#define   VERSION            51013          // Eventually will be deprecated. 
+#define   SEMVERSION         "5.1.13.dev0"  // Semantic Version
```

Comment:
@goanpeca, I think splitting these up into major minor and patch is good but I'm not sure that containing this into a `struct` is necessary.